docs/_static/parallelizer.dot

```diff
@@ -0,0 +1,19 @@
+digraph parallelizer {
```

Comment:
This is bases on a DSL : https://graphviz.org/doc/info/lang.html
Can be used to describe arbitrary graphs, that can be rendered.

Reply:
This is pretty cool, but you have to create it by hand? is there a way to automatically generate them? ()

Reply:
Will need to check something like that. Sounds quite radical, but I am not sure if there is something like that (cursory search)

Reply:
mmm cool, I guess we would have to add stuff to the docstring for any tool to autogenerate something like that. Probably not worth it at this point. But we can leave the ones you created for reference and over time add more and experiment. If we realize they go out of sync fast and hard to maintain, we just remove them all.